src/material-experimental/mdc-button-toggle/harness/button-toggle-harness-filters.ts

```diff
@@ -0,0 +1,12 @@
+/**
+ * @license
+ * Copyright Google LLC All Rights Reserved.
+ *
+ * Use of this source code is governed by an MIT-style license that can be
+ * found in the LICENSE file at https://angular.io/license
+ */
+
+export type ButtonToggleHarnessFilters = {
+  label?: string | RegExp,
```

Comment:
I think we should use `text` for button-like elements since the text is _inside_ the component (vs checkbox and radio where the label accompanies the actual component)